design/scss/custom/_faq.scss

```
@@ -1,6 +1,21 @@
 #faq {
+    .accordion-body {
+        font-weight: bold;
+    }
+
+    .accordion-button {
+        background-color: $primary;
+        color:$white;
+    }
+
+    .accordion-button:focus{
+        border-color: inherit;
+        box-shadow: none;
+    }
+    
 	.collapsed {
 		color: $white;
-		background-color: $primary;
+		background-color: $acordion-collapsed-color;
```

```suggestion
		background-color: $accordion-collapsed-color;
```

minor spelling fix to go along with [this suggestion](https://github.com/donategifts/donategifts/pull/651/files#r1300673075)